drivers/espi/host_subs_npcx.c

```diff
@@ -293,6 +293,10 @@ static void host_kbc_obe_isr(const void *arg)
 
 	LOG_DBG("%s: kbc status 0x%02x", __func__, inst_kbc->HIKMST);
 
+#if defined(CONFIG_ESPI_NPCX_i8042_KBC_AUX_VWIRE_IRQ_WORKAROUND)
```

Comment:
Non-blocking nit.  It is preferred to use `if IS_ENABLED(CONFIG...))` when possible.  I think you can convert all #ifdefs in this file.